### ISSUE TITLE
Clean up Help menu

### ISSUE THREAD

**Issue by hinerm** [imagej/imagej-legacy#115]
We should consider the following:

* Remove `Switch to Modern Mode` as a legacy plugin
* Remove either `Update...` or `Update fiji` since they do the same thing right now.

**Comment by ctrueden** [comment]
Yes, instead of "Switch to Modern Mode", perhaps there should be a command somewhere to switch user interfaces (or maybe launch a second UI). But it's very low priority.

And agreed that `Update Fiji` should have gone away a long time ago. There are many obsolete plugins still shipped with Fiji. We can remove them all in one fell swoop, but we need to send a big warning to fiji-devel first. And then be prepared to defend our decision.
